### DISCUSSION TITLE
Does skywalking agent support pulsar 2.9.x ?

### DISCUSSION THREAD

**Discussion by mgyboom** [apache/skywalking#10733] [Q&A] [answered]
According to this document, you can see that pulsar 2.2.x ~ 2.9.x is supported：
https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/supported-list/
Download from here：
https://www.apache.org/dyn/closer.cgi/skywalking/java-agent/8.15.0/apache-skywalking-java-agent-8.15.0.tgz
Unpack tgz，find pulsar plugins in ./skywalking-agent/plugins：
apm-pulsar-2.2-2.7-plugin-8.15.0.jar
apm-pulsar-2.8.x-plugin-8.15.0.jar
apm-pulsar-common-8.15.0.jar

**Comment by wu-sheng** [comment, marked as answer]
From Pulsar community submissions, 2.9.0 has been verified basically.

https://github.com/apache/skywalking-java/blob/main/test/plugin/scenarios/pulsar-scenario/support-version.list

**Reply by ZeweiLiuu** [reply]
Is there any follow-up support plan for the pulsar plugin? Currently, the official version 4.x is already available.

**Reply by wu-sheng** [reply]
No one from Pulsar community updated since 2.9. Maybe you are better to ask at Pulsar community?

**Reply by ZeweiLiuu** [reply]
![05294DE8](https://github.com/user-attachments/assets/733c1791-1744-4a74-a3ae-612579c5207a)
